internal/renderers/skia/lib.rs

```diff
@@ -68,7 +68,13 @@ pub struct SkiaRenderer {
 }
 
 impl SkiaRenderer {
-    /// Creates a new renderer is associated with the provided window adapter.
+    /// Creates a new renderer for the given window/display handle. The provided size
+    /// must match the size of the underlying native window.
+    ///
+    /// Subsequent calls to [`Self::render()`] will render into the window.
+    ///
+    /// Safety: It is the caller's responsibility to ensure that the underlying native
+    /// window exists and remains valid for the life-time of the SkiaRenderer.
     pub fn new(
         window_handle: raw_window_handle::WindowHandle<'_>,
         display_handle: raw_window_handle::DisplayHandle<'_>,
```

Comment:
And this means unfortunately that we are stuck with v0.5 of raw-window-handle for the remaining of Slint 1.x
(while 0.6 is about to be released)

Reply:
Any indications for when that would be? I see at least two blocking issues.

(but the NSWindow changes make me realize that we can prepare our C++ API already perhaps)